mezzanine/project_template/deploy/gunicorn.conf.py.template

```diff
@@ -1,7 +1,8 @@
 from __future__ import unicode_literals
 import multiprocessing
 
-bind = "127.0.0.1:%(gunicorn_port)s"
-workers = multiprocessing.cpu_count() * 2 + 1
+bind = "unix:%(proj_path)s/gunicorn.sock"
+workers = 2
```

Comment:
In my experience you usually have access to 8 or 16 cores but as little as 512Mb or 1Gb of RAM in a VPS. When running multiple sites you end up with a bunch of workers that eat up your memory. However, single sites probably don't have this problem. I'll add the setting as you suggest as it seems a nice compromise.